util/fmtutil/format_test.go

```diff
@@ -15,11 +15,13 @@ package fmtutil
 
 import (
 	"bytes"
+	"math"
 	"testing"
+	"time"
 
-	"github.com/stretchr/testify/require"
-
+	dto "github.com/prometheus/client_model/go"
 	"github.com/prometheus/prometheus/prompb"
+	"github.com/stretchr/testify/require"
```

Comment:
prompb is a "local" package, goimports is run with :
```
goimports -w -local github.com/prometheus/prometheus util/fmtutil/format_test.go 
```

```suggestion
	dto "github.com/prometheus/client_model/go"
	"github.com/stretchr/testify/require"
	
	"github.com/prometheus/prometheus/prompb"
```